help/help.adoc

```diff
@@ -264,129 +324,77 @@ This procedure allows users to delete a Virtual Network using Azure Services.
-|Network Security Rule |Name of the Network security Rule which is to be deleted. (Required)
-|Network Security Group |Name of the Network security Group in which network security rule is present. (Required)
-|Resource Group Name |The name of the resource group in which network security group is present. (Required)
-|JobStep Timeout |Timeout for the step execution in minutes. Blank means no timeout.
-|===
-
-image::cloudbees-common::cd-plugins/ec-azure/parameters/delete-network-security-rule.png[image]
-
-[[Delete-Subnet]]
+|Operating System Type |Required only for user images.
+|Resource Group Name |The name of the resource group in which VM is to be created. (Required)
+|Location |Location of the resource. (Required)
+|Storage Account a|Name of storage account. (Required)
 
+If this storage account resides in a resource group different from VM itself, it must be specified via -rgn prefix argument.
 
-[[Delete_Subnet]]
-=== Delete subnet
+For example, `-rgn ResourceGroupName StorageAccountName`
+|Storage Container |Name of the storage container. (Required)
+|Virtual Network a|Name of an existing Virtual Network for VM to be connected to.
 
-This procedure allows users to delete a subnet.
+If this virtual network resides in a resource group different from VM itself, it must be specified via -rgn prefix argument.
 
-[cols=",",options="header",]
-|===
-|Parameter |Description
-|Configuration |The name of the configuration which holds all the connection information for Windows Azure. (Required)
-|Subnet |Name of the subnet which is to be deleted. (Required)
-|Virtual Network |Name of the virtual network in which subnet is present. (Required)
-|Resource Group Name |The name of the resource group in which subnet is present. (Required)
+For example, `-rgn ResourceGroupName VirtualNetworkName`
+|Subnet |Name of an existing subnet within a Virtual Network. Requires Virtual Network to be set.
+|Create Public IP Address? |Create Public IP address for the deployed servers?
+|Resource Pool |If you would like to add {PRODUCT} resources for each Azure instance being created, enter the resource pool name for the new resources. If left blank no resource will be created.
+|Resource Port |If you specify a resource pool name in 'Resource Pool' field, this is the port that will be used when creating the resource. If no value is specified, port 7800 will be used by default when creating the resource.
+|Resource workspace |If you specify a resource pool name, this is the workspace that will be assigned to the created resources.
+|Resource Zone Name |Created resource will belong to the specified zone. Zone 'default' is used by default.
+|Machine size |Size of the machine. For example, Standard_D2_v2. Refer to link:https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs[Virtual machines sizes] for reference.
+|Results Location |Location where the output properties will be saved. If not specified, thsi will default to '/myJob/Azure/deployed'.
```

Comment:
Figured we can update this too, since already editing. (Also generally recommend using [present tense](https://engineering.beescloud.com/docs/beewrite/latest/style#_using_present_tense_and_future_tense_appropriately) rather than future tense in most cases, but probably not worth updating all other instances in this file since this plugin will soon be unsupported.)
```suggestion
|Results Location |Location where the output properties is saved. If not specified, this defaults to '/myJob/Azure/deployed'.
```

Reply:
updated